bin/varnishd/proxy/cache_proxy_proto.c

```diff
@@ -423,6 +429,15 @@ vpx_proto2(const struct worker *wrk, struct req *req)
 		VSL(SLT_ProxyGarbage, req->sp->vxid, "PROXY2: header length mismatch");
 		return (0);
 	}
+	if (tlv_len) {
+		SES_Reserve_xport_priv(req->sp, &up);
+		WS_Reserve(req->sp->ws, 2 + tlv_len);
```

Comment:
check for error

Reply:
fixed, also remove WS_MarkOverflow in vmod_proxy.c (already set in WS_Reserve)